src/bunit.core/Asserting/AssertionMethodAttribute.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Bunit.Asserting
+{
+	/// <summary>
+	/// Add this attribute to assertion methods to indicate to
+	/// 3rd party analyzers that the method is an assertion method.
+	/// See more here: https://jira.sonarsource.com/browse/RSPEC-3413
```

Comment:
I suggest to use link to official source, even when it was not released yet. We'll deprecate the Jira RSPEC project in near future.
```suggestion
	/// See more here: https://rules.sonarsource.com/csharp/RSPEC-2699
```